test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -0,0 +1,113 @@
+import * as path from 'path';
+import { DAPP_URL, largeDelayMs, openDapp, unlockWallet } from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+
+/**
+ * Default options for setting up Multichain E2E test environment
+ */
+export const DEFAULT_OPTIONS = {
+  dapp: true,
+  dappPaths: [
+    path.join(
+      '..',
+      '..',
+      'node_modules',
+      '@metamask',
+      'test-dapp-multichain',
+      'build',
+    ),
+  ],
+};
+
+/**
+ * Unlocks a wallet and provides extension id for dapp to connect to wallet extension.
+ *
+ * @param {Driver} driver - E2E test driver {@link Driver}, wrapping the Selenium WebDriver.
+ * @param extensionId - Extension identifier for web dapp to interact with wallet extension.
+ */
+export async function openDappAndConnectWallet(
```

Comment:
```suggestion
export async function openMultichainDappAndConnectWalletWithExternallyConnectable(
```
[nit]

Probably helps others to be very specific, since `connect` is now a more overloaded concept - previously it has only ever meant granting a dapp permissions via the connect UI flow.

Reply:
resolved in [396ec6f](https://github.com/MetaMask/metamask-extension/pull/29298/commits/396ec6ffcf728182695a52a8f42b9c4b72122933)